docs/user/whats-new.asciidoc

```diff
@@ -64,38 +64,31 @@ image::images/currierlocation.png[Example of currier locations in relations to s
+**Lens** now supports a new line interpolation option for line charts. Step curve line charts help users see clearly when data changes at irregular intervals, reducing misinterpretation. When creating visualizations select **Line** in the visualization type dropdown. Then use the **Visual options** menu to choose step in the **Line interpolation** dropdown.
 
 [role="screenshot"]
 image::images/stepcurve.gif[An short video displaying the three new chart options]
 
 [discrete]
 ==== Significance (significant terms)
 
-This new ranking method in the top values breakdown displays data by significance (significant terms). It permits us to get the terms that are important and likely the most critical for a given query. It’s a powerful way to find anomalies in your data with multiple use cases. Read more {ref}/search-aggregations-bucket-significantterms-aggregation.html[here].
-This is not just based on the most popular terms in a set. When calculating significant terms, Elasticsearch looks for significant changes in popularity of a specific term between two sets of data. A foreground set (the data returned by a query) and a background set (the data in our indices). This is supported on all charts. 
+This new ranking method in the top values breakdown displays data by significance (significant terms). It helps you identify the key and most crucial terms for a specific query. It’s a powerful way to find anomalies in your data with multiple use cases. Read more {ref}/search-aggregations-bucket-significantterms-aggregation.html[here].
+This is not just based on the most popular terms in a set. When calculating significant terms, {es} looks for significant changes in popularity of a specific term between two sets of data. A foreground set (the data returned by a query) and a background set (the data in our indices). This is supported on all charts. 
 
 [role="screenshot"]
 image::images/significant-terms.png[Example of a donut chart organized by significant terms, width=70%]
 
 [discrete]
 ==== Tag cloud
 
-You can now create a word cloud in **Lens** using Tag cloud, which is currently in technical preview. When creating new visualizations, select **Tag cloud** in the visualization type dropdown. From the available fields drag and drop the desired field into the workspace to visualize keywords. 
+preview:[] You can now create a word cloud in **Lens** using Tag cloud, which is currently in technical preview. When creating new visualizations, select **Tag cloud** in the visualization type dropdown. From the available fields drag and drop the desired field into the workspace to visualize keywords. 
```

Comment:
Because you have preview at the beginning, you don't need "which is currently in technical preview".

Command after "From the available fields,"